### PR TITLE
Feat(assembly): use delegator in calc_voting_power

### PR DESCRIPTION
**Pull request by maksk7**

### CODE REVIEW

File: contracts/assembly/src/migration.rs [hunk 1]
```diff
@@ -82,3 +115,41 @@ pub(crate) fn migrate_proposals_to_v111(deps: &mut DepsMut, cfg: &Config) -> Std
...
+        proposal_voting_period: cfg_v100.proposal_voting_period,
+        proposal_effective_delay: cfg_v100.proposal_effective_delay,
+        proposal_expiration_period: cfg_v100.proposal_expiration_period,
+        proposal_required_deposit: cfg_v100.proposal_required_deposit,
+        proposal_required_quorum: cfg_v100.proposal_required_quorum,
+        proposal_required_threshold: cfg_v100.proposal_required_threshold,
+        whitelisted_links: cfg_v100.whitelisted_links,
+    };
+
+    if let Some(vxastro_token_addr) = msg.vxastro_token_addr {
```
[the remark this answers is not in the document]
Comment (maksk7) [reply contracts/assembly/src/migration.rs:139]: vxastro_token_addr may already be set in the config, so it has to change only if msg has vxastro_token_addr

File: contracts/assembly/src/contract.rs [hunk 2]
```diff
@@ -67,6 +69,7 @@ pub fn instantiate(
     let mut config = Config {
         xastro_token_addr: addr_validate_to_lower(deps.api, &msg.xastro_token_addr)?,
         vxastro_token_addr: None,
+        voting_escrow_delegator_addr: None,
```
[the remark this answers is not in the document]
Comment (maksk7) [reply contracts/assembly/src/contract.rs]: donte, thanks

**Review by P-Yevhenii — APPROVED** [review]

File: contracts/assembly/src/migration.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (epanchee) [reply contracts/assembly/src/migration.rs:139]: Yes, that's true. Anyway, cfg.voting_escrow_delegator_addr can be set directly when creating new Config structure

[the remark this answers is not in the document]
Comment (maksk7) [reply contracts/assembly/src/migration.rs:142]: True, updated, thanks

File: contracts/assembly/src/migration.rs [hunk 3]
```diff
@@ -125,7 +125,10 @@ pub(crate) fn migrate_config_to_130(
     let mut cfg = Config {
```
Comment (epanchee) [inline contracts/assembly/src/migration.rs:125]: I think mut is not needed anymore?

Comment (maksk7) [reply contracts/assembly/src/migration.rs:125]: We need it to set cfg.vxastro_token_addr at 143

**Review by ShadoySV — APPROVED** [review]

**Pull request merged**